src/DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -271,14 +271,11 @@ public function process(ContainerBuilder $container): void
             }
         }
 
-        if (!empty($customTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setTypes', [$customTypes]);
-        }
-        if (!empty($customNotMappedTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setNotMappedTypes', [$customNotMappedTypes]);
-        }
+        $staticMapperDefinition = new Definition(StaticTypeMapper::class, [
+            '$types' => $customTypes,
+            '$notMappedTypes' => $customNotMappedTypes
+        ]);
+        $container->setDefinition('graphql.type_mapper', $staticMapperDefinition);
```

Comment:
A new way to declare the DI definition cannot be overridden by the project (because the compiler pass fully redefines the DI service), but in case of graphqlite immutability, probably it is OK move.